src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -33,6 +33,12 @@ type ScreenProps = StackScreenProps<StackParamList, Screens.FiatDetailsScreen>
 
 type Props = ScreenProps
 
+export const SUPPORTED_FIAT_ACCOUNT_TYPES = new Set<FiatAccountType>([FiatAccountType.BankAccount])
+
+export const SUPPORTED_FIAT_ACCOUNT_SCHEMAS = new Set<FiatAccountSchema>([
```

Comment:
We also have https://github.com/valora-inc/wallet/blob/f57f9ac03e62b2e5b34a77460c1b967689a49ef2/src/navigator/types.tsx#L55, maybe these should be together?